build-dist.xml

```diff
@@ -659,7 +659,7 @@ java -jar ${app.server.weblogic.dir}/${app.server.weblogic.jar.name} -ignoreSysP
 				<matches pattern="10\.\d+\.\d+" string="${app.server.tomcat.version}" />
 			</and>
 			<then>
-				<var name="transformer.agent" value="-javaagent:${CATALINA_HOME}/../tools/portal-tools-jakarta-ee-transformer/com.liferay.portal.tools.jakarta.ee.transformer.jar" />
+				<var name="transformer.agent" value="\&quot;-javaagent:${CATALINA_HOME}/../tools/portal-tools-jakarta-ee-transformer/com.liferay.portal.tools.jakarta.ee.transformer.jar\&quot;" />
```

Comment:
Hi @shuyangzhou ,
This is to fix https://testray.liferay.com/#/project/35392/routines/45357/build/184890961/case-result/184999457, CATALINA_HOME will break when it see "bun dle"